src/vmm/src/arch/aarch64/fdt.rs

```diff
@@ -258,6 +275,22 @@ fn create_chosen_node(
     Ok(())
 }
 
+fn create_vmgenid_node(fdt: &mut FdtWriter, vmgenid: &Option<VmGenId>) -> Result<(), FdtError> {
+    if let Some(vmgenid_info) = vmgenid {
+        let compatible = "microsoft,vmgenid";
+
+        let vmgenid = fdt.begin_node("vmgenid")?;
+        fdt.property_string("compatible", compatible)?;
+        fdt.property_array_u64("reg", &[vmgenid_info.guest_address.0, 4096])?;
```

Comment:
Is `4096` specified somewhere in the vmgenid? Maybe we should have a constant for it? 
Also having 4KB memory for 128bit integer seems a bit much, can this be reduced somehow?